rest-api-spec/test/indices.recovery/10_basic.yaml

```diff
@@ -1,9 +1,6 @@
 ---
 "Indices recovery test":
 
-  - skip:
-      features: gtelte
-
```

Comment:
Yes, I deleted that on master already. Should also maybe delete the gtelte on 1.x branch?